pyroengine/engine/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+from .pyronear_predictor import PyronearPredictor
+from .pyronear_engine import PyronearEngine
```

Comment:
let's rename the files "predictor.py" and "engine.py" to avoid complicated names (sine the package is already named pyroengine)